ethereum/core/src/main/java/tech/pegasys/teku/core/operationvalidators/ProposerSlashingStateTransitionValidator.java

```diff
@@ -45,7 +45,9 @@
                 ProposerSlashingInvalidReason.PROPOSER_INDICES_DIFFERENT),
         () ->
             check(
-                !Objects.equals(proposerSlashing.getHeader_1(), proposerSlashing.getHeader_2()),
+                !Objects.equals(
+                    proposerSlashing.getHeader_1().getMessage(),
+                    proposerSlashing.getHeader_2().getMessage()),
```

Comment:
(nit) We can simplify:
```suggestion
                !Objects.equals(
                    header1.getMessage(),
                    header2.getMessage()),
```

Reply:
Done.